sass/modules/_buttons.scss

```diff
@@ -190,44 +190,22 @@ label.btn {
     content: $icon-circle-blank;
   }
 
-  input.checked + & {
```

Comment:
I'm not sure what was adding it and not entirely sure it was an actual class... seems to be like the pseudo selector where it reacts to the state of the item, firefox was just getting in a state (possibly after a re-render) that it was not clearing the .checked state when you select a different option.

This updates it to only use the :checked selector only which works fine IE9+/chrome/firefox/etc.